eclipse-scout-core/src/index.ts

```diff
@@ -940,6 +951,7 @@ export * from './tabbox/SimpleTabArea';
 export * from './tabbox/SimpleTabAreaModel';
 export * from './tabbox/SimpleTabAreaEventMap';
 export * from './tabbox/SimpleTabAreaLayout';
+export * from './calendar/CalendarsPanel';
```

Comment:
JavaScript does interpret the imports from top to bottom. because I need the TreeBox in the CalendarsPanel, i have to place the import below the one of the TreeBox